unittests/resources/settings.py

```diff
@@ -90,7 +90,7 @@
     'environments': [
         {
             'name': 'PrgEnv-gnu',
-            'modules': [{'name': 'PrgEnv-gnu', 'collection': False}],
+            'modules': [{'name': 'PrgEnv-gnu', 'collection': False, 'path': None}],
```

Comment:
If this is to be accepted, for which I'm in favor, we should update the config schema (making sure that the `additionalProperties` is `False` of this, which apparently I have forgotten to put in the current version of the module spec. We should also update the documentation.

Reply:
I don't think that the `path` is needed here.